fxmanifest.lua

```diff
@@ -1,13 +1,31 @@
-fx_version 'cerulean'
-game 'gta5'
-
-version '1.0.0'
-repository 'https://github.com/Qbox-project/qbx-properties'
-description 'Hopefully one day a feature rich property system'
-
-shared_script 'config.lua'
-client_script 'client.lua'
-server_script 'server.lua'
-
-lua54 'yes'
-use_experimental_fxv2_oal 'yes'
+fx_version 'cerulean'
+game 'gta5'
+
+version '0.0.1'

```

Comment:
I think 1.0.0 would mean the script is fully functionning and released, so that's why i changed it to avoid confusion, but yeah could definitely remove it